docs/README.md

```diff
@@ -86,7 +86,7 @@ the source code and be captured by revision control (currently git). This way th
 includes the version of the documentation that is relevant regardless of which version or release
 you have checked out or downloaded.
 
-In this directory you will find text files formatted using Markdown, with an ".md" suffix. You can
+In this directory you will find text files formatted using Markdown, with a ".md" suffix. You can
```

Comment:
I think it's really no biggie. we call it MD suffix, and an can be legitimate?

Reply:
Thanks for your review, I'm just trying to find an open source project and  contribute.   I will contribute more.